lisa/util/process.py

```diff
@@ -23,11 +24,30 @@ class ExecutableResult:
     stdout: str
     stderr: str
     exit_code: Optional[int]
+    cmd: Union[str, List[str]]
     elapsed: float
 
     def __str__(self) -> str:
         return self.stdout
 
+    def assert_exit_code(
+        self, expected_exit_code: int = 0, message: str = ""
+    ) -> AssertionBuilder:
+        display_message = f"get unexpected exit code on cmd {self.cmd}"
+        if message:
+            display_message += f", {message}"
```

Comment:
The user inputted message should be more important. So it should be put before the generated message.